inc/namespace.php

```diff
@@ -34,6 +34,12 @@ function bootstrap() {
  * @return Group[] Sorted list of groupse
  */
 function get_documentation() : array {
+	static $documentation;
```

Comment:
Why not `static $docs`?

Reply:
I didn't want to conflict with the local `$docs` variable, incase that had some unintentional impact.

Reply:
In this case, it'd avoid the `$documentation = $docs` line, so would simplify a bit. There's only one return path from the rest of the function, so IMO we should do it.